service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storage/StorageAccountKeyProviderUnitTest.java

```diff
@@ -0,0 +1,50 @@
+package bio.terra.workspace.service.resource.controlled.cloud.azure.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import bio.terra.workspace.app.configuration.external.AzureConfiguration;
+import bio.terra.workspace.common.BaseUnitTest;
+import bio.terra.workspace.service.crl.CrlService;
+import bio.terra.workspace.service.workspace.AzureCloudContextService;
+import bio.terra.workspace.service.workspace.model.AzureCloudContext;
+import com.azure.resourcemanager.storage.StorageManager;
+import com.azure.resourcemanager.storage.models.StorageAccount;
+import com.azure.resourcemanager.storage.models.StorageAccountKey;
+import com.azure.resourcemanager.storage.models.StorageAccounts;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+
+public class StorageAccountKeyProviderUnitTest extends BaseUnitTest {
+
+  @Test
+  public void getStorageAccountKey() {
+    var azureCloudContextService = mock(AzureCloudContextService.class);
```

Comment:
Same thing with these mocks - having the dependencies marked with `@MockBean` outside the test case itself and setup code in a shared `@BeforeEach` method could help clarify what's setup and what's test logic